todo_app/test_viewmodel.py

```diff
@@ -0,0 +1,41 @@
+from todo_app.templates.view_model import ViewModel
+from todo_app.data.item import Item
+
+def test_view_can_ret_todo_items():
+    items= [Item('1234', 'start', 'todo')]
+    item_view_model = ViewModel(items)
+
+    todo_items = item_view_model.todo_items
+
+    assert len (todo_items) == 1
```

Comment:
You're not really checking that any filtering is happening. Because the whole list of items is being returned.

It would be more useful to do something like this:

```python
todo_item = Item('1234', 'start', 'todo')
done_item = Item('1234', 'start', 'done')
items = [todo_item, done_item]
item_view_model = ViewModel(items)

todo_items = item_view_model.todo_items

assert todo_items = [todo_item]
```

But as mentioned in another comment, the status should actually be "To Do" rather than "todo".